src/gui/optionsdialog.ui

```diff
@@ -3054,6 +3054,13 @@ Specify an IPv4 or IPv6 address. You can specify &quot;0.0.0.0&quot; for any IPv
                     </property>
                    </widget>
                   </item>
+                  <item>
+                   <widget class="QCheckBox" name="checkSecureCookie">
+                    <property name="text">
+                     <string>Enable cookie Secure flag (warning: must have working HTTPS)</string>
```

Comment:
Would it make sense move the warning to tooltip?
Also I find "working" to be a little redundant, how about just: "Require HTTPS"?

Reply:
Since the most recent change, I don't think a tooltip is needed at all. I will just make it so that the option is greyed out if HTTPS is disabled like @glassez  suggested below. And yeah, I'll change the description to "Requires HTTPS"